zeppelin-splicemachine/pom.xml

```diff
@@ -0,0 +1,126 @@
+        <commons.pool2.version>2.3</commons.pool2.version>
+        <commons.logging.version>1.1.1</commons.logging.version>
+        <slf.logging.version>1.7.10</slf.logging.version>
+        <log4j.version>2.11.1</log4j.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.zeppelin</groupId>
+            <artifactId>zeppelin-interpreter</artifactId>
+            <version>${zeppelin.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.zeppelin</groupId>
+            <artifactId>zeppelin-jdbc</artifactId>
+            <version>${zeppelin.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.splicemachine</groupId>
+            <artifactId>db-client</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.mockrunner</groupId>
+            <artifactId>mockrunner-jdbc</artifactId>
+            <version>${mockrunner.jdbc.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-dbcp2</artifactId>
+            <version>${commons.dbcp2.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-pool2</artifactId>
+            <version>${commons.pool2.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+            <version>${commons.logging.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+            <version>${slf.logging.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>${slf.logging.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>${log4j.version}</version>
+        </dependency>
+    </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-enforcer-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <artifactId>maven-resources-plugin</artifactId>
```

Comment:
The plugin is missing a version. And causes a warning
```
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-resources-plugin is missing. @ com.splicemachine:zeppelin-splicemachine-0.8.0:[unknown-version], /Users/user/Github/spliceengine/zeppelin-splicemachine/pom.xml, line 107, column 21
```
This is  version that was used in assembly.xml
```
<version>3.1.0</version>
```

Reply:
I think we presently have our poms as children of 'our parent', aka the spliceengine-parent; that is the structure all of our poms currently have. It would be antithetical to have a pom in this project taht was a child of a third-party pom, i think?

I agree that the transitive dependencies now have to be resolved, though. Perhaps this pom just declares them, or if appropriate we edit teh spliceengine-parent pom.xml to provide the needed version variable(s)?

Reply:
@MurrayBrown making `interpreter-parent` the parent of this might be inconsistent with the rest of the modules in this project, but it should not be problematic. It is not generally antithetical to specify a parent that is different from the project in which it is listed as a module. There is no strict need for the relationship to be 2-way. In this case, making `interpreter-parent` the parent will likely resolve all of the build problems for this module.